app/assets/stylesheets/base/typography.scss

```diff
@@ -96,44 +95,52 @@ body {
   h5,
   h6 {
     font-family: var(--ff-sans-serif);
-    margin: var(--content-rythm) 0;
-    margin-top: calc(var(--content-rythm) * 2);
+    margin: calc(var(--content-rhythm) / 2) 0;
     padding: 0;
+    font-weight: var(--fw-bold);
+    line-height: var(--lh-tight);
   }
 
   h1 {
-    font-size: 1.85em;
-    line-height: 1.25em;
-    font-weight: 400;
+    font-size: 2em;
```

Comment:
Something I notice (and I hope it isn't an incorrect observation but let me know if it is) is that when I resize the viewport to the smallest size, the `h` texts stay the same size as when it's on XL viewport. I'm wondering if it might be more ideal for these to scale relative to viewport because something on a phone doesn't need to see a really really big `h1`; all they need to know is that it's the most emphasize title relative to everything else.

I've heard of using `rem`s as one way to approach this and I would love to hear how you're thinking about this.

Reply:
> Something I notice (and I hope it isn't an incorrect observation but let me know if it is) is that when I resize the viewport to the smallest size, the h texts stay the same size as when it's on XL viewport.

I'll check, maybe something is broken.

> I've heard of using rems as one way to approach this

`rem`s are a bit different - these are relative to the font size defined in the `body` element. These are much more useful in scenarios like this for simple layouts and designs, especially when *everything* can be scalable based on font-size (including all kinda of sizes for elements, paddings, margins, etc. etc)...
These won't be very useful in our case (here, where we define article typography) - but we do use `rem`s in other areas, where appropriate.

Reply:
I see -- thanks for that explanation :)